src/components/HomePage/HothDescription.js

```diff
@@ -84,11 +84,11 @@ export default function HothDescription() {
 					</Grid>
 					<Grid item sm={10} md={6}>
 						<Typography variant='body1' style={{ paddingBottom: isSmall ? theme.spacing(2) : 0 }}>
-							Hack off the Hill (HOTH 8) is a{' '}
+							Hack on the Hill (HOTH 9) is a{' '}
 							<strong>beginner-friendly</strong>
 							{' '}hackathon designed to be your first hackathon experience.
-							Due to the ongoing global pandemic,
-							HOTH 8 will be virtual and span 2 days!
+							Due to recent spread of the Omicron variant,
```

Comment:
```suggestion
							Due to recent spread of Covid variants,
```
In case somehow another variant becomes more relevant before HOTH begins rip